cwl/src/test/scala/cwl/CommandOutputExpressionSpec.scala

```diff
@@ -11,30 +11,49 @@ import eu.timepit.refined.string.MatchesRegex
 import ExpressionEvaluator._
 import cats.data.Validated.Valid
 import wdl.types.WdlIntegerType
-import wdl.values.{WdlGlobFile, WdlInteger, WdlString}
-import wom.expression.PlaceholderIoFunctionSet
+import wdl.values.{WdlGlobFile, WdlInteger, WdlString, WdlValue}
+import wom.expression.{IoFunctionSet, PlaceholderIoFunctionSet}
+
+import scala.concurrent.Future
+import scala.util.Try
 
 class CommandOutputExpressionSpec extends FlatSpec with Matchers {
 
   behavior of "CommandOutputExpression"
 
+  def ioFunctionSet(data: String) =
+    new IoFunctionSet {
+      override def readFile(path: String) = Future.successful(data)
+
+      override def writeFile(path: String, content: String) = ???
```

Comment:
I'd prefer real throws here (ala "this method should never have been called by this test"), to show that this is the final state of the test case

Reply:
ok. 

for the record I think a better solution here is to pass in a `readFile: String => IO[String]` function to `evaluateValue` which would make this mock unnecessary.